api/src/main/java/org/openmrs/api/impl/UserServiceImpl.java

```diff
@@ -751,21 +755,36 @@ public User getUserByActivationKey(String activationKey) {
 	}
 	
 	/**
+	 * @throws MessageException
 	 * @see org.openmrs.api.UserService#setUserActivationKey(org.openmrs.User)
-	 * 
 	 */
 	@Override
 	public User setUserActivationKey(User user) {
 		String token = RandomStringUtils.randomAlphanumeric(20);
 		long time = System.currentTimeMillis() + getValidTime();
 		String hashedKey = Security.encodeString(token);
-		String activationKey = hashedKey+":"+time;
+		String activationKey = hashedKey + ":" + time;
 		LoginCredential credentials = dao.getLoginCredential(user);
 		credentials.setActivationKey(activationKey);	
 		dao.setUserActivationKey(credentials);	
-			
-		//Send Email with unhashed  activation key and Date:
 		
+		Properties props = new Properties();
```

Comment:
The file isn't supposed to be a properties file, otherwise you might as well move everything to the messages.properties file which already supports internationalization anyways and get rid of this file.